scripts/eessi_sets.yml

```diff
@@ -50,3 +50,43 @@ eessi_sets:
           - macos-aarch64
           - macos-x86_64
       - name: sys-process/numactl
+  - name: eessi-2025.01
```

Comment:
@bedroge bump to `2025.02`?

Reply:
Done in https://github.com/EESSI/gentoo-overlay/pull/108/commits/e53017a1c0d6261be12b358bb5e5142da2ac54cb.